source/opencascade/boundary_lib.cc

```diff
@@ -248,9 +248,7 @@ namespace OpenCASCADE
     curve(curve_adaptor(sh)),
     tolerance(tolerance),
     length(shape_length(sh))
-  {
-    Assert(spacedim == 3, ExcNotImplemented());
-  }
+  {}
```

Comment:
Is `spacedim<2` allowed?

Reply:
No, it is not instantiated for `spacedim < 2`. I shall put back an Assert with spacedim>=2